README.md

```diff
@@ -212,7 +233,8 @@ The `js/translations.xx.json` file, where `xx` is the BCP47 language code, needs
   },
   "labels": {
     "N/A": "",
-    "%feature in %spec": ""
+    "%feature in %spec": "",
+    "Polyfills": ""
```

Comment:
Change "Polyfills" to lowercase for consistency? The right side (i.e., the "value" in the name/value pair) can be capitalized.

BTW, should it be added to `translations.json`?

Reply:
Ah, naming is hard, as usual ;)

In practice, these translation files contain two different types of "translations":
1. Mappings between internal types and labels: the left side is the ID of the type, and the right side is the translated string that is to be rendered. That's what the `sections`, `columns` and `implstatus` properties contain in current translation files. Obviously, the mapping needs to appear somewhere for English too.
2. Translations of English labels: the left side is the rendered English label, and the right side is the translated label. That's what the `labels`, `groups`, `specifications` and `features` properties contain. By definition, there is no need to specify the English translation anywhere for such a label: the English translation is the label itself.

The second point is usually how translation gets done in applications: the code uses English strings, typically wrapped in `tr` calls. Some tool gets used to parse the code and extract these strings, which are then used as IDs in translation files. That's how `gettext` works for instance.

I'm not sure what is the best practice to handle mappings. I found it useful to put these mappings in a separate `translations.json` file for English (noting the name of the file is not exactly correct, but `mappings.json` did not sound very good either), and to continue to use the same mechanism in translation files in other languages. It may be better practice to stick to pure translation files, i.e. to have a flat structure in `translations.zh.json`, with the left side being the English label (and not the internal type ID) to translate. At the same time, having a little bit of structure helps set the context for the strings to be translated, and I don't think we'll have a hell of a lot of strings to translate, expect for group names and specification titles, which we probably want to put aside in any case, as they can be useful in other contexts than roadmap documents.

Anyway, case matters for string keys that appear in the `labels` property, because they are not internal IDs: they are the rendered strings.